packages/salesforcedx-vscode-lwc/typings/open/index.d.ts

```diff
@@ -0,0 +1,71 @@
+/// <reference types="node"/>
+import { ChildProcess } from 'child_process';
+
+// IMPORTANT NOTE:
```

Comment:
basically this file can be removed and the tsconfig change can be undone once the version of typescript is updated.